src/components/slicks/SlideShow.jsx

```diff
@@ -43,10 +47,12 @@ export const SlideShow = forwardRef((props, ref) => {
                 {slides?.map((slide, index) => (
                     <div key={index} className="!flex !justify-between !items-center">
                         <FollowCard
```

Comment:
e nên có thêm tham số trạng thái card disasble | not yet followed| followed để hiển thị UI , thêm sự kiện onClick button ( (cái này mình sẽ gọi tời các function khác nhau). như e viết hiện tại chỉ có thể ap dụng cho 3 loại giả sử sau này có thêm 2 loại card nữa e lại phải vào sửa code tiếp